src/ol/render/canvas/Replay.js

```diff
@@ -456,9 +476,15 @@ class CanvasReplay extends VectorContext {
   }
 
   /**
-   * FIXME empty description for jsdoc
+   * @return {Object} the serializable instructions.
```

Comment:
Change `Object` to `SerializableInstructions`, and maybe return `undefined`values for `textStates`, `fillStates` and `strokeStates` ?